src/System.Net.Sockets/src/System/Net/Sockets/SocketPal.Unix.cs

```diff
@@ -703,6 +703,8 @@ public static bool TryCompleteSendTo(SafeSocketHandle socket, IList<ArraySegment
         public static bool TryCompleteSendTo(SafeSocketHandle socket, ReadOnlySpan<byte> buffer, IList<ArraySegment<byte>> buffers, ref int bufferIndex, ref int offset, ref int count, SocketFlags flags, byte[] socketAddress, int socketAddressLen, ref int bytesSent, out SocketError errorCode)
         {
             bool successfulSend = false;
+            long start = Environment.TickCount64;
```

Comment:
Does this have any measurable impact on throughput?

Reply:
I'm not sure how to measure it. I modified the code so we only hit this when underlying socket is blocking and timeout is set. So it should not be common case.